scripts/download_MPI_cmip_data.sh

```diff
@@ -0,0 +1,1049 @@
+#!/bin/bash
```

Comment:
Perhaps add some documentation on how this file was obtained/generated?

Reply:
@Imme1992 can you give me the website link?